docs/users.adoc

```diff
@@ -0,0 +1,122 @@
+// vim: tw=79
+= Users
+:toc:
+
+== Create Admin User
+
+Generate a default Tendrl admin user.
+
+Run the following command from the root of the project.
+
+----------
+bundle exec rake etcd:load_admin
+----------
+
+The command will generate a admin user with a password.
+
+== Create User
+
+Create a new user.
+
+Sample Request
+
+----------
+curl -H 'Content-Type: application/json' -H 'Authorization: Bearer
+d03ebb195dbe6385a7caeda699f9930ff2e49f29c381ed82dc95aa642a7660b8' 
+-XPOST -d '{"name":"Tom Hardy", "username":"thardy", "email":"thardy@tendrl.org", "role":"admin", "password":"pass1234", "password_confirmation":"pass1234"}'
```

Comment:
@mbukatov This pull request is already merged. I've filed issue yesterday https://github.com/Tendrl/api/issues/106